### PR TITLE
Generate the src/*.rdf files from a template

### LINKED ISSUE #67: un-revert the rdf templating revert

### PR DESCRIPTION
**Pull request by pdehaan**
Fixes #67

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,11 +29,15 @@
...
-    "url": "https://github.com/mozilla/universal-search-addon"
+    "url": "git+https://github.com/mozilla/universal-search-addon.git"
   },
   "scripts": {
     "build": "gulp build",
     "lint": "gulp lint",
+    "release": "npm run release-patch",
+    "release-major": "npm version major && npm run build",
+    "release-minor": "npm version minor && npm run build",
+    "release-patch": "npm version patch && npm run build",
```
Comment (pdehaan) [inline package.json]: We talked about it earlier, but I added the `release` scripts to this PR instead of doing a separate one.

**Comment by jaredhirsch** [comment]
@pdehaan mind rebasing?

**Comment by pdehaan** [comment]
@6a68 Rebased.

**Comment by jaredhirsch** [comment]
I'll rebase it this time...

File: README.md [hunk 2]
```diff
@@ -37,11 +37,13 @@ To work around this:
 1. Restart the browser, and you're in business.
 
 ## Release process:
-  1. Manually bump the version number in `src/install.rdf` and `src/update.rdf`.
-  1. Bump the version number in package.json using `npm version patch`. This will generate a git tag, too.
-  1. Zip up a new addon: `gulp build`.
+  1. Bump the package.json version number and create an XPI using one of the following commands:
```
Comment (6a68) [inline README.md]: Mind rewording to "Use one of the following commands to bump the package.json version number and create an XPI"?

Comment (6a68) [inline README.md]: Meh. Forget it

**Comment by jaredhirsch** [comment]
Works for me locally. Thanks @pdehaan!

**Pull request merged**